lib/resources/example_json/dstu2_examples_slot.rb

```diff
@@ -40,709 +40,719 @@ module Resources
 
     DSTU2_SLOT_BUNDLE ||= {
       "resourceType": "Bundle",
-      "id": "0b37fa7f-7711-4e8c-990e-158a02fc92c4",
+      "id": "b51b12e5-0387-423d-a3ab-83a7e214dd89",
       "type": "searchset",
       "link": [
-        {
-          "relation": "self",
-          "url": "https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Slot?slot-type=http%3A%2F%2Fsnomed.info%2Fsct%7C394581000&schedule.actor=Practitioner%2F2578010&start=2016&_count=5"
```

Comment:
Why did we remove the schedule.actor here?

Reply:
Adding to this, it was probably because you used the actual response. However, it may be good to keep the schedule.actor values in the response (even if it is not actually there) so people can see we return it in responses if it's there. The point of this disclaimer if saying "hey this might not be totally accurate" lol

Reply:
I was going off of what the Jira said to do, as leaving schedule.actor in the request was returning an empty bundle: "The current search example returns an empty bundle. Removing the "schedule.actor" param from the request URL fixes this.
https://fhir.cerner.com/millennium/dstu2/scheduling/slot/#example"

Reply:
Let's update the schedule.actor to one that returns at least one result. The results are already dated in 2019, so it's fine to just have a response that has 2019 results.

Also, the use of a schedule.actor param is highly-requested by most scheduling consumers of the API. They care about viewing all the slots that a given Practitioner may have as much as they want to view _all_ Practitioners' slots. For that reason too, we should keep the param.

Reply:
Good points, found one and added back here: https://github.com/cerner/fhir.cerner.com/commit/c98315ef854eea8353f363c5c6f04f3c52df4618

Reply:
Left comments